docs/tutorials/cargo_tests/baked/build.rs

```diff
@@ -0,0 +1,34 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use icu::locid::langid;
+use icu_datagen::CldrLocaleSubset;
+use icu_datagen::SourceData;
+use icu_datagen::prelude::*;
+use std::path::PathBuf;
+
+fn main() {
+    // TODO(#3046): We shouldn't need to do this since it worked in 1.0
+    let mod_directory = PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("baked_data");
+    if mod_directory.exists() {
+        std::fs::remove_dir_all(&mod_directory).unwrap();
+    }
+
+    icu_datagen::datagen(
+        Some(&[langid!("ru")]),
+        &[icu::plurals::provider::CardinalV1Marker::KEY],
+        &SourceData::default()
+            .with_cldr_for_tag(SourceData::LATEST_TESTED_CLDR_TAG, CldrLocaleSubset::Modern)
+            .expect("Source data should download successfully"),
+        vec![icu_datagen::Out::Module {
+            // For more information on these options, see:
+            // <https://icu4x.unicode.org/doc/icu_datagen/enum.Out.html#variant.Module>
+            mod_directory,
+            pretty: true,
```

Comment:
`pretty: true` requires `rustfmt`, which might trip up clients if they run this on CI. Let's use `false` in all examples.

Reply:
ok